instrumentation/azure-core/azure-core-1.19/javaagent/build.gradle.kts

```diff
@@ -29,6 +29,7 @@ dependencies {
   // Ensure no cross interference
   testInstrumentation(project(":instrumentation:azure-core:azure-core-1.14:javaagent"))
```

Comment:
would there be any objections if I removed azure-core-1.14 and 1.19? 

I'm pulling download stats to confirm (will take a bit), but the assumption is that the usage of 3+ year old package would be quite low.

Reply:
we're super conservative on dropping old versions from the javaagent: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/VERSIONING.md#javaagent-instrumentation

Reply:
cool, then I'll keep them, but will try to separate common code from multiple versions

Reply:
copy-paste is ok, it's pretty small instrumentation and easy to diff between the different versions as needed for now